docs-src/typescript/how-to-log-from-a-workflow-in-typescript.md

```diff
@@ -41,16 +41,18 @@ Explicitly declaring a Sink's interface is optional, but is useful for ensuring
 
 <!--SNIPSTART typescript-logger-sink-interface-->
 
-[packages/test/src/workflows/definitions.ts](https://github.com/temporalio/sdk-typescript/blob/master/packages/test/src/workflows/definitions.ts)
+[sinks/src/workflows.ts](https://github.com/temporalio/samples-typescript/blob/master/sinks/src/workflows.ts)
 
 ```ts
-import type { Sinks } from '@temporalio/workflow';
+import { LoggerSinks, proxySinks, Sinks } from '@temporalio/workflow';
```

Comment:
Should we change the title/description of the PR to include that there are TS changes too? Even if automatically obtained, this PR does more than add Python testing sections, it also syncs other code.

Reply:
I suggest making that work in a separate PR